# Cluster5-2 Analysis

## Proteins: 

ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BICRA, BICRAL, CDCA7, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1

# Chromatin Remodeling and Transcription Regulation System

## Summary

The human protein system analyzed here consists of 20 proteins, most of which are involved in chromatin remodeling and transcription by RNA polymerase II. The proteins are also associated with various cellular components and complexes, such as the SWI/SNF complex, brahma complex, npBAF complex, nBAF complex, and GBAF complex. The system plays a role in multiple biological processes, including cell differentiation, cell population proliferation, and DNA repair. Based on the cellular location and function of these proteins, we propose the name "Chromatin Remodeling and Transcription Regulation System" for this protein system.

## References

1. Clapier, C. R., & Cairns, B. R. (2009). The biology of chromatin remodeling complexes. Annual review of biochemistry, 78, 273-304. [DOI: 10.1146/annurev.biochem.77.062706.153223](https://doi.org/10.1146/annurev.biochem.77.062706.153223)

2. Narlikar, G. J., Sundaramoorthy, R., & Owen-Hughes, T. (2013). Mechanisms and functions of ATP-dependent chromatin-remodeling enzymes. Cell, 154(3), 490-503. [DOI: 10.1016/j.cell.2013.07.011](https://doi.org/10.1016/j.cell.2013.07.011)

3. Hargreaves, D. C., & Crabtree, G. R. (2011). ATP-dependent chromatin remodeling: genetics, genomics and mechanisms. Cell research, 21(3), 396-420. [DOI: 10.1038/cr.2011.32](https://doi.org/10.1038/cr.2011.32)

4. Kadoch, C., & Crabtree, G. R. (2015). Mammalian SWI/SNF chromatin remodeling complexes and cancer: Mechanistic insights gained from human genomics. Science advances, 1(5), e1500447. [DOI: 10.1126/sciadv.1500447](https://doi.org/10.1126/sciadv.1500447)

5. Mashtalir, N., D'Avino, A. R., Michel, B. C., Luo, J., Pan, J., Otto, J. E., ... & Kadoch, C. (2018). Modular organization and assembly of SWI/SNF family chromatin remodeling complexes. Cell, 175(5), 1272-1288. [DOI: 10.1016/j.cell.2018.09.032](https://doi.org/10.1016/j.cell.2018.09.032)

## Gene Feature Summary: 

| Feature | Number of Genes | Genes |
| --- | --- | --- |
| chromatin remodeling | 19 | ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BICRA, BICRAL, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  transcription by RNA polymerase II | 19 | ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BICRA, BICRAL, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| chromatin | 19 | ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BICRA, BICRAL, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| protein binding | 18 | ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BICRA, BICRAL, CDCA7, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCE1 |
| SWI/SNF complex | 18 | ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BICRA, BICRAL, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  mitotic metaphase/anaphase transition | 17 | ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  G0 to G1 transition | 17 | ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  G1/S transition of mitotic cell cycle | 17 | ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  double-strand break repair | 17 | ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  nucleotide-excision repair | 17 | ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  cell differentiation | 15 | ACTL6A, ARID1A, ARID1B, ARID2, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| nucleoplasm | 15 | ACTL6A, ARID1A, ARID1B, ARID2, CDCA7, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  DNA-templated transcription | 14 | ACTL6A, ARID1A, ARID1B, BICRA, BICRAL, CDCA7, SMARCA2, SMARCA4, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  T cell differentiation | 14 | ACTL6A, ARID1A, ARID1B, ARID2, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  myoblast differentiation | 14 | ACTL6A, ARID1A, ARID1B, ARID2, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| nucleus | 13 | ACTL6A, ARID1A, BICRA, CDCA7, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| transcription coactivator activity | 12 | ACTL6A, ARID1A, ARID1B, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  stem cell population maintenance | 12 | ACTL6A, ARID1A, BCL7A, BCL7B, BICRA, BICRAL, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCD1, SMARCE1 |
| nervous system development | 11 | ACTL6A, ARID1A, ARID1B, DPF1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD3 |
| brahma complex | 11 | ACTL6A, ARID1A, ARID1B, SMARCA2, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| npBAF complex | 11 | ACTL6A, ARID1A, ARID1B, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD3, SMARCE1 |
| nBAF complex | 11 | ARID1A, ARID1B, DPF1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD3, SMARCE1 |
|  cell population proliferation | 10 | ACTL6A, BCL7A, BCL7B, BICRA, BICRAL, CDCA7, SMARCA2, SMARCA4, SMARCC1, SMARCD1 |
| kinetochore | 10 | ACTL6A, ARID2, PBRM1, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCE1 |
| nuclear matrix | 10 | ACTL6A, ARID2, PBRM1, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCE1 |
| RSC-type complex | 10 | ACTL6A, ARID2, PBRM1, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCE1 |
| nucleosome disassembly | 10 | ARID1A, ARID2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| negative regulation of cell differentiation | 9 | ACTL6A, BCL7A, BCL7B, BICRA, BICRAL, SMARCA2, SMARCA4, SMARCC1, SMARCD1 |
| GBAF complex | 9 | ACTL6A, BCL7A, BCL7B, BICRA, BICRAL, SMARCA2, SMARCA4, SMARCC1, SMARCD1 |
| bBAF complex | 8 | ARID1A, ARID1B, SMARCA2, SMARCA4, SMARCB1, SMARCC2, SMARCD2, SMARCE1 |
| nucleosomal DNA binding | 7 | ACTL6A, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD2, SMARCE1 |
| protein-containing complex | 7 | ACTL6A, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD2, SMARCE1 |
| DNA binding | 7 | ARID1A, ARID1B, ARID2, PBRM1, SMARCA2, SMARCA4, SMARCB1 |
| chromatin binding | 5 | ACTL6A, PBRM1, SMARCA2, SMARCD1, SMARCE1 |
| transcription initiation-coupled chromatin remodeling | 5 | ARID1A, ARID1B, SMARCA4, SMARCB1, SMARCD1 |
| negative regulation of DNA-templated transcription | 5 | BCL7A, SMARCA2, SMARCA4, SMARCC2, SMARCE1 |
| negative regulation of cell population proliferation | 4 | ARID2, PBRM1, SMARCA2, SMARCB1 |
| neural retina development | 3 | ACTL6A, SMARCA4, SMARCD3 |
| plasma membrane | 3 | ACTL6A, ARID1B, ARID2 |
| nuclear receptor binding | 3 | ARID1A, SMARCD3, SMARCE1 |
| apoptotic process | 3 | BCL7B, CDCA7, DPF1 |
| nuclear chromosome | 3 | PBRM1, SMARCB1, SMARCE1 |
| histone binding | 3 | SMARCA2, SMARCC1, SMARCC2 |
| intracellular membrane-bounded organelle | 3 | SMARCA2, SMARCB1, SMARCD1 |
| transcription coregulator activity | 3 | SMARCD1, SMARCD2, SMARCD3 |
| DNA-templated transcription | 3 | SMARCD1, SMARCD2, SMARCD3 |
|  double-strand break repair via homologous recombination | 2 | ACTL6A, ARID2 |
| nucleosome binding | 2 | ARID1A, ARID1B |
| cytosol | 2 | ARID1B, CDCA7 |
| metal ion binding | 2 | ARID2, DPF1 |
| cytoplasm | 2 | DPF1, SMARCC1 |
| helicase activity | 2 | SMARCA2, SMARCA4 |
| ATP binding | 2 | SMARCA2, SMARCA4 |
| ATP-dependent activity, acting on DNA | 2 | SMARCA2, SMARCA4 |
| hydrolase activity | 2 | SMARCA2, SMARCA4 |
| ATP-dependent chromatin remodeler activity | 2 | SMARCA2, SMARCA4 |
| negative regulation of transcription by RNA polymerase II | 2 | SMARCA2, SMARCA4 |
| negative regulation of cell growth | 2 | SMARCA2, SMARCA4 |
| RNA polymerase I core promoter sequence-specific DNA binding | 2 | SMARCA4, SMARCB1 |
| transcription coregulator binding | 2 | SMARCA4, SMARCD3 |
| p53 binding | 2 | SMARCA4, SMARCB1 |
| RNA binding | 2 | SMARCA4, SMARCE1 |
| Tat protein binding | 2 | SMARCA4, SMARCB1 |
| RNA polymerase I preinitiation complex assembly | 2 | SMARCA4, SMARCB1 |
| positive regulation by host of viral transcription | 2 | SMARCA4, SMARCB1 |
|  DNA-binding transcription factor activity | 2 | SMARCA4, SMARCB1 |
|  transcription of nucleolar large rRNA by RNA polymerase I | 2 | SMARCA4, SMARCB1 |
|  glucose mediated signaling pathway | 2 | SMARCA4, SMARCB1 |
| fibrillar center | 2 | SMARCA4, SMARCB1 |
| nucleolus | 2 | SMARCA4, SMARCB1 |
| XY body | 2 | SMARCB1, SMARCC1 |
| signaling receptor binding | 2 | SMARCD1, SMARCD3 |

