# Cluster1-5 Analysis

## Proteins: 

ACADM, ACADVL, ACO2, ACOT2, AKR7A2, AKR7A3, ALDH6A1, ARG2, ARMC1, BID, CIAPIN1, CPNE3, CPOX, DCXR, DDX28, DNAJC11, DPYSL4, DTYMK, ECHS1, FH, GAA, GALE, GCA, GCLM, GPD2, GPX1, GRPEL1, HADH, HIBCH, HNRNPLL, HSDL2, IARS2, ICAM3, IDH2, IDH3A, LMO4, MAP4K2, ME2, MPST, NLN, NUDT9, OXCT1, PAPSS2, PNPT1, PPA2, PPCS, PPIF, PPM1B, PUS1, RAB14, RAB7A, SLC25A14, SPRYD4, TAGLN, TRMT10C, UCHL3, USP46

## Mitochondrial Protein Interaction System

### Summary

The human protein system analyzed in this report consists of 37 proteins primarily localized in the mitochondrion, with many of them involved in protein binding, cytosol, and mitochondrial matrix. The system is involved in various biological processes such as tricarboxylic acid cycle, electron transfer activity, electron transport chain, fatty acid beta-oxidation, and response to xenobiotic stimulus. Based on the cellular location and function, we propose the name "Mitochondrial Protein Interaction System" for this protein system.

### Analysis

The Mitochondrial Protein Interaction System is predominantly composed of proteins localized in the mitochondrion (37 proteins), with a significant number of them involved in protein binding (35 proteins) and cytosol (28 proteins) [1]. The system is involved in several essential biological processes, including the tricarboxylic acid cycle (4 proteins: IDH2, IDH3A, FH, ACO2), electron transfer activity (4 proteins: CIAPIN1, ME2, AKR7A2, AKR7A3), and electron transport chain (4 proteins: CIAPIN1, ME2, AKR7A2, AKR7A3) [2]. These processes are crucial for cellular energy production and metabolism.

Another significant function of the Mitochondrial Protein Interaction System is fatty acid beta-oxidation (3 proteins: ACADM, ECHS1, HADH), which is essential for breaking down fatty acids to generate energy [3]. Additionally, the system is involved in the response to xenobiotic stimulus (4 proteins: GPX1, OXCT1, GCLM, HADH), which is crucial for detoxification and protection against harmful substances [4].

Given the critical roles of these proteins in cellular energy production, metabolism, and detoxification, it is essential to understand their interactions and functions. While some of the proteins in this system are well-characterized, others may represent novel members of known complexes or even form a novel complex altogether. Further experimental validation and analysis of protein-protein interactions within this system will provide valuable insights into the functional roles of these proteins and their potential involvement in human diseases.

### References

[1] Ashburner, M., Ball, C. A., Blake, J. A., Botstein, D., Butler, H., Cherry, J. M., ... & Sherlock, G. (2000). Gene ontology: tool for the unification of biology. Nature genetics, 25(1), 25-29.

[2] Pagliarini, D. J., Calvo, S. E., Chang, B., Sheth, S. A., Vafai, S. B., Ong, S. E., ... & Mootha, V. K. (2008). A mitochondrial protein compendium elucidates complex I disease biology. Cell, 134(1), 112-123.

[3] Wanders, R. J., & Waterham, H. R. (2006). Biochemistry of mammalian peroxisomes revisited. Annual review of biochemistry, 75, 295-332.

[4] Kensler, T. W., Wakabayashi, N., & Biswal, S. (2007). Cell survival responses to environmental stresses via the Keap1-Nrf2-ARE pathway. Annual review of pharmacology and toxicology, 47, 89-116.

## Gene Feature Summary: 

| Feature | Number of Genes | Genes |
| --- | --- | --- |
| mitochondrion | 37 | IDH2, HIBCH, CIAPIN1, ACOT2, HSDL2, PUS1, ALDH6A1, NUDT9, PPA2, ACADM, ECHS1, IARS2, IDH3A, GPD2, ME2, GPX1, TRMT10C, PNPT1, OXCT1, FH, ACADVL, ACO2, GRPEL1, DTYMK, ARMC1, SLC25A14, AKR7A2, ARG2, BID, CPNE3, CPOX, DDX28, DNAJC11, HADH, MPST, PPIF, RAB7A |
| protein binding | 35 | CIAPIN1, ACOT2, HNRNPLL, PAPSS2, NUDT9, DPYSL4, ECHS1, IDH3A, TRMT10C, PNPT1, ICAM3, FH, ACADVL, GRPEL1, ARMC1, AKR7A2, AKR7A3, ARG2, BID, CPNE3, DCXR, DNAJC11, GCA, GCLM, LMO4, MAP4K2, MPST, PPIF, PPM1B, RAB14, RAB7A, SPRYD4, TAGLN, UCHL3, USP46 |
| cytosol | 28 | IDH2, ACOT2, PAPSS2, DPYSL4, PPCS, GPX1, PNPT1, FH, ACO2, DTYMK, ARMC1, AKR7A2, AKR7A3, BID, CPNE3, CPOX, DCXR, DDX28, GALE, GCA, GCLM, MPST, NLN, PPM1B, RAB14, RAB7A, UCHL3, USP46 |
| mitochondrial matrix | 24 | IDH2, HIBCH, ACOT2, PUS1, ALDH6A1, NUDT9, PPA2, ACADM, ECHS1, IARS2, IDH3A, ME2, GPX1, TRMT10C, PNPT1, OXCT1, FH, ACADVL, ACO2, GRPEL1, ARG2, HADH, MPST, PPIF |
| cytoplasm | 19 | CIAPIN1, PUS1, ACADM, PPCS, GPX1, PNPT1, FH, DTYMK, AKR7A2, AKR7A3, ARG2, CPNE3, CPOX, GCA, HADH, MAP4K2, PPIF, TAGLN, UCHL3 |
| nucleus | 15 | HNRNPLL, PUS1, ACADM, IDH3A, PPCS, GPX1, TRMT10C, PNPT1, FH, DTYMK, CPNE3, DCXR, PPM1B, SPRYD4, USP46 |
| extracellular exosome | 13 | IDH2, NUDT9, ICAM3, FH, AKR7A2, AKR7A3, CPNE3, DCXR, GAA, GCA, MPST, RAB14, RAB7A |
| identical protein binding | 12 | ACADM, PPCS, TRMT10C, PNPT1, OXCT1, ACADVL, GRPEL1, AKR7A3, DCXR, GALE, HADH, MPST |
| nucleoplasm | 11 | CIAPIN1, PUS1, ALDH6A1, TRMT10C, ACADVL, GRPEL1, CPNE3, DDX28, DNAJC11, HADH, UCHL3 |
| plasma membrane | 9 | ICAM3, SLC25A14, CPNE3, DCXR, GAA, GCA, NLN, RAB14, RAB7A |
| RNA binding | 7 | HNRNPLL, PUS1, ALDH6A1, TRMT10C, PNPT1, CPNE3, DDX28 |
| membrane | 6 | HSDL2, HNRNPLL, BID, GAA, PPIF, PPM1B |
| ATP binding | 6 | PAPSS2, IARS2, PPCS, DTYMK, DDX28, MAP4K2 |
| metal ion binding | 6 | ME2, ARMC1, CPNE3, LMO4, NLN, USP46 |
| nucleolus | 5 | CIAPIN1, ACADVL, CPNE3, DDX28, PPM1B |
| protein homodimerization activity | 5 | PPCS, GRPEL1, CPOX, GALE, GCA |
| magnesium ion binding | 4 | IDH2, PPA2, IDH3A, PPM1B |
| tricarboxylic acid cycle | 4 | IDH2, IDH3A, FH, ACO2 |
| electron transfer activity | 4 | CIAPIN1, ME2, AKR7A2, AKR7A3 |
| electron transport chain | 4 | CIAPIN1, ME2, AKR7A2, AKR7A3 |
| mitochondrial intermembrane space | 4 | CIAPIN1, PNPT1, CPOX, NLN |
| mitochondrial inner membrane | 4 | GPD2, ACADVL, SLC25A14, CPOX |
| response to xenobiotic stimulus | 4 | GPX1, OXCT1, GCLM, HADH |
| NAD binding | 3 | IDH2, IDH3A, ME2 |
| carbohydrate metabolic process | 3 | IDH2, IDH3A, AKR7A2 |
| isocitrate metabolic process | 3 | IDH2, IDH3A, ACO2 |
| branched-chain amino acid catabolic process | 3 | HIBCH, ALDH6A1, ECHS1 |
| synapse | 3 | HNRNPLL, PPA2, MPST |
| tRNA binding | 3 | PUS1, IARS2, TRMT10C |
| liver development | 3 | ACADM, ACO2, MPST |
| fatty acid beta-oxidation | 3 | ACADM, ECHS1, HADH |
| mitochondrial membrane | 3 | ACADM, ACADVL, RAB7A |
| intracellular membrane-bounded organelle | 3 | ME2, GAA, RAB14 |
| mitochondrial nucleoid | 3 | TRMT10C, ACADVL, DDX28 |
| response to activity | 3 | OXCT1, GCLM, HADH |
| Golgi apparatus | 3 | AKR7A2, AKR7A3, RAB7A |
| apoptotic mitochondrial changes | 3 | BID, GCLM, PPIF |
| lysosome | 3 | GAA, RAB14, RAB7A |
| lysosomal membrane | 3 | GAA, RAB14, RAB7A |
| NADP metabolic process | 2 | IDH2, DCXR |
| peroxisome | 2 | IDH2, HSDL2 |
| valine catabolic process | 2 | HIBCH, ALDH6A1 |
| iron ion binding | 2 | CIAPIN1, ACO2 |
| 4 iron, 4 sulfur cluster binding | 2 | CIAPIN1, ACO2 |
| negative regulation of apoptotic process | 2 | CIAPIN1, PPIF |
| molecular_function | 2 | HSDL2, SPRYD4 |
| biological_process | 2 | HSDL2, SPRYD4 |
| mRNA processing | 2 | HNRNPLL, PUS1 |
| fatty-acyl-CoA binding | 2 | ALDH6A1, ACADVL |
| phosphorylation | 2 | PAPSS2, DTYMK |
| cell junction | 2 | NUDT9, CPNE3 |
| protein serine/threonine phosphatase activity | 2 | PPA2, PPM1B |
|  mitochondrial membrane potential | 2 | PPA2, BID |
| acyl-CoA dehydrogenase activity | 2 | ACADM, ACADVL |
| flavin adenine dinucleotide binding | 2 | ACADM, ACADVL |
|  gluconeogenesis | 2 | ACADM, NLN |
| response to cold | 2 | ACADM, ACADVL |
| fatty acid beta-oxidation using acyl-CoA dehydrogenase | 2 | ACADM, ACADVL |
| response to starvation | 2 | ACADM, OXCT1 |
| calcium ion binding | 2 | GPD2, GCA |
| malate metabolic process | 2 | ME2, FH |
| temperature homeostasis | 2 | GPX1, ACADVL |
| response to hormone | 2 | GPX1, OXCT1 |
| cellular response to oxidative stress | 2 | GPX1, PNPT1 |
| neuron apoptotic process | 2 | GPX1, BID |
| cellular response to glucose stimulus | 2 | GPX1, GCLM |
| negative regulation of release of cytochrome c from mitochondria | 2 | GPX1, PPIF |
| mitochondrial RNA 5'-end processing | 2 | TRMT10C, PNPT1 |
|  cellular senescence | 2 | PNPT1, ARG2 |
| response to nutrient | 2 | OXCT1, GCLM |
|  cold-induced thermogenesis | 2 | FH, HADH |
| epithelial cell differentiation | 2 | ACADVL, TAGLN |
| protein folding | 2 | GRPEL1, PPIF |
| intracellular protein transport | 2 | GRPEL1, RAB14 |
| cellular response to growth factor stimulus | 2 | DTYMK, CPNE3 |
| mitochondrial outer membrane | 2 | ARMC1, BID |
| aldo-keto reductase (NADP) activity | 2 | AKR7A2, AKR7A3 |
| cellular aldehyde metabolic process | 2 | AKR7A2, AKR7A3 |
| manganese ion binding | 2 | ARG2, PPM1B |
| innate immune response | 2 | ARG2, MAP4K2 |
|  release of cytochrome c from mitochondria | 2 | BID, PPIF |
| protein serine/threonine kinase activity | 2 | CPNE3, MAP4K2 |
|  cell migration | 2 | CPNE3, LMO4 |
| cellular response to calcium ion | 2 | CPNE3, PPIF |
| azurophil granule membrane | 2 | CPNE3, GAA |
| glucose metabolic process | 2 | DCXR, GAA |
| tertiary granule membrane | 2 | GAA, RAB14 |
| extracellular region | 2 | GCA, NLN |
| Golgi membrane | 2 | MAP4K2, RAB14 |
| GTPase activity | 2 | RAB14, RAB7A |
| GTP binding | 2 | RAB14, RAB7A |
| GDP binding | 2 | RAB14, RAB7A |
| intracellular transport | 2 | RAB14, RAB7A |
| phagosome maturation | 2 | RAB14, RAB7A |
| late endosome | 2 | RAB14, RAB7A |
| phagocytic vesicle | 2 | RAB14, RAB7A |
| cysteine-type deubiquitinase activity | 2 | UCHL3, USP46 |
| ubiquitin-dependent protein catabolic process | 2 | UCHL3, USP46 |
| protein deubiquitination | 2 | UCHL3, USP46 |

