# Cluster4-5 Analysis

## Proteins: 

ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BCL7C, BICRA, BICRAL, BOD1L1, BRD9, C17orf49, CDCA7, CHD8, CTDSPL2, DPF1, DPF2, INTS9, KPNA2, KPNA6, NIPBL, NUP50, PBRM1, PDIK1L, PHAX, RAMAC, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1, SRCAP, SRRT, SS18L1

# Chromatin Remodeling and Transcription Regulation System

## Summary

The human protein system analyzed in this study is primarily involved in chromatin remodeling and transcription regulation. The majority of the proteins are located in the nucleoplasm and nucleus, with several proteins also found in the cytosol and other cellular compartments. The system is enriched in proteins that are part of the SWI/SNF complex, which plays a crucial role in chromatin remodeling and transcription regulation. Additionally, many proteins are involved in transcription by RNA polymerase II, cell differentiation, and various cellular processes such as cell cycle transitions, DNA repair, and stem cell maintenance. The proposed name for this system is the Chromatin Remodeling and Transcription Regulation System, reflecting its primary cellular location and function.

## References

1. Clapier, C. R., & Cairns, B. R. (2009). The biology of chromatin remodeling complexes. Annual review of biochemistry, 78, 273-304. [DOI: 10.1146/annurev.biochem.77.062706.153223](https://doi.org/10.1146/annurev.biochem.77.062706.153223)

2. Narlikar, G. J., Sundaramoorthy, R., & Owen-Hughes, T. (2013). Mechanisms and functions of ATP-dependent chromatin-remodeling enzymes. Cell, 154(3), 490-503. [DOI: 10.1016/j.cell.2013.07.011](https://doi.org/10.1016/j.cell.2013.07.011)

3. Hargreaves, D. C., & Crabtree, G. R. (2011). ATP-dependent chromatin remodeling: genetics, genomics and mechanisms. Cell research, 21(3), 396-420. [DOI: 10.1038/cr.2011.32](https://doi.org/10.1038/cr.2011.32)

4. Kadoch, C., & Crabtree, G. R. (2015). Mammalian SWI/SNF chromatin remodeling complexes and cancer: Mechanistic insights gained from human genomics. Science Advances, 1(5), e1500447. [DOI: 10.1126/sciadv.1500447](https://doi.org/10.1126/sciadv.1500447)

5. Ronan, J. L., Wu, W., & Crabtree, G. R. (2013). From neural development to cognition: unexpected roles for chromatin. Nature Reviews Genetics, 14(5), 347-359. [DOI: 10.1038/nrg3413](https://doi.org/10.1038/nrg3413)

## Gene Feature Summary: 

| Feature | Number of Genes | Genes |
| --- | --- | --- |
| protein binding | 35 | KPNA6, C17orf49, KPNA2, DPF2, BRD9, CHD8, PDIK1L, NUP50, SRRT, SRCAP, BOD1L1, SS18L1, NIPBL, PHAX, INTS9, RAMAC, CTDSPL2, ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BICRA, BICRAL, CDCA7, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCE1 |
| nucleoplasm | 31 | KPNA6, C17orf49, KPNA2, DPF2, BRD9, CHD8, PDIK1L, NUP50, SRRT, SRCAP, BOD1L1, SS18L1, NIPBL, PHAX, INTS9, CTDSPL2, ACTL6A, ARID1A, ARID1B, ARID2, CDCA7, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  transcription by RNA polymerase II | 28 | KPNA6, DPF2, BRD9, CHD8, SRCAP, SS18L1, NIPBL, BCL7C, CTDSPL2, ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BICRA, BICRAL, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| nucleus | 24 | KPNA6, KPNA2, DPF2, BRD9, CHD8, SRCAP, SS18L1, NIPBL, PHAX, INTS9, RAMAC, ACTL6A, ARID1A, BICRA, CDCA7, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| chromatin remodeling | 24 | DPF2, BRD9, CHD8, SRCAP, BCL7C, ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BICRA, BICRAL, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| chromatin | 24 | DPF2, BRD9, CHD8, NIPBL, BCL7C, ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BICRA, BICRAL, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| SWI/SNF complex | 21 | DPF2, BRD9, BCL7C, ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, BICRA, BICRAL, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  mitotic metaphase/anaphase transition | 19 | DPF2, BCL7C, ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  G0 to G1 transition | 19 | DPF2, BCL7C, ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  G1/S transition of mitotic cell cycle | 19 | DPF2, BCL7C, ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  double-strand break repair | 19 | DPF2, BCL7C, ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  nucleotide-excision repair | 19 | DPF2, BCL7C, ACTL6A, ARID1A, ARID1B, ARID2, BCL7A, BCL7B, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  DNA-templated transcription | 18 | CHD8, SRRT, SRCAP, SS18L1, ACTL6A, ARID1A, ARID1B, BICRA, BICRAL, CDCA7, SMARCA2, SMARCA4, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  stem cell population maintenance | 15 | DPF2, BRD9, BCL7C, ACTL6A, ARID1A, BCL7A, BCL7B, BICRA, BICRAL, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCD1, SMARCE1 |
|  cell differentiation | 15 | ACTL6A, ARID1A, ARID1B, ARID2, DPF1, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| transcription coactivator activity | 14 | SRCAP, SS18L1, ACTL6A, ARID1A, ARID1B, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  T cell differentiation | 14 | ACTL6A, ARID1A, ARID1B, ARID2, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
|  myoblast differentiation | 14 | ACTL6A, ARID1A, ARID1B, ARID2, PBRM1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| nBAF complex | 13 | DPF2, SS18L1, ARID1A, ARID1B, DPF1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD3, SMARCE1 |
| nervous system development | 12 | DPF2, ACTL6A, ARID1A, ARID1B, DPF1, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD3 |
|  cell population proliferation | 12 | BRD9, BCL7C, ACTL6A, BCL7A, BCL7B, BICRA, BICRAL, CDCA7, SMARCA2, SMARCA4, SMARCC1, SMARCD1 |
| DNA binding | 11 | C17orf49, CHD8, SRRT, SRCAP, ARID1A, ARID1B, ARID2, PBRM1, SMARCA2, SMARCA4, SMARCB1 |
| negative regulation of cell differentiation | 11 | BRD9, BCL7C, ACTL6A, BCL7A, BCL7B, BICRA, BICRAL, SMARCA2, SMARCA4, SMARCC1, SMARCD1 |
| GBAF complex | 11 | BRD9, BCL7C, ACTL6A, BCL7A, BCL7B, BICRA, BICRAL, SMARCA2, SMARCA4, SMARCC1, SMARCD1 |
| kinetochore | 11 | SS18L1, ACTL6A, ARID2, PBRM1, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCE1 |
| brahma complex | 11 | ACTL6A, ARID1A, ARID1B, SMARCA2, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| npBAF complex | 11 | ACTL6A, ARID1A, ARID1B, SMARCA2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD3, SMARCE1 |
| cytosol | 10 | KPNA6, C17orf49, KPNA2, DPF2, SS18L1, NIPBL, PHAX, INTS9, ARID1B, CDCA7 |
| protein-containing complex | 10 | CHD8, SRRT, SRCAP, ACTL6A, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD2, SMARCE1 |
| nuclear matrix | 10 | ACTL6A, ARID2, PBRM1, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCE1 |
| RSC-type complex | 10 | ACTL6A, ARID2, PBRM1, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCE1 |
| nucleosome disassembly | 10 | ARID1A, ARID2, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD1, SMARCD2, SMARCD3, SMARCE1 |
| bBAF complex | 8 | ARID1A, ARID1B, SMARCA2, SMARCA4, SMARCB1, SMARCC2, SMARCD2, SMARCE1 |
| chromatin binding | 7 | CHD8, NIPBL, ACTL6A, PBRM1, SMARCA2, SMARCD1, SMARCE1 |
| negative regulation of DNA-templated transcription | 7 | CHD8, NIPBL, BCL7A, SMARCA2, SMARCA4, SMARCC2, SMARCE1 |
| nucleosomal DNA binding | 7 | ACTL6A, SMARCA4, SMARCB1, SMARCC1, SMARCC2, SMARCD2, SMARCE1 |
| RNA binding | 6 | KPNA2, SRRT, PHAX, RAMAC, SMARCA4, SMARCE1 |
| negative regulation of transcription by RNA polymerase II | 5 | DPF2, CHD8, NIPBL, SMARCA2, SMARCA4 |
| apoptotic process | 5 | DPF2, BCL7C, BCL7B, CDCA7, DPF1 |
| intracellular membrane-bounded organelle | 5 | DPF2, NIPBL, SMARCA2, SMARCB1, SMARCD1 |
| ATP binding | 5 | CHD8, PDIK1L, SRCAP, SMARCA2, SMARCA4 |
| histone binding | 5 | CHD8, SRCAP, SMARCA2, SMARCC1, SMARCC2 |
| transcription initiation-coupled chromatin remodeling | 5 | ARID1A, ARID1B, SMARCA4, SMARCB1, SMARCD1 |
| cytoplasm | 4 | KPNA2, SRRT, DPF1, SMARCC1 |
| ATP-dependent chromatin remodeler activity | 4 | CHD8, SRCAP, SMARCA2, SMARCA4 |
| negative regulation of cell population proliferation | 4 | ARID2, PBRM1, SMARCA2, SMARCB1 |
| protein import into nucleus | 3 | KPNA6, KPNA2, NUP50 |
| membrane | 3 | KPNA6, KPNA2, SMARCA4 |
| metal ion binding | 3 | DPF2, ARID2, DPF1 |
| lysine-acetylated histone binding | 3 | DPF2, BRD9, SMARCA4 |
| p53 binding | 3 | CHD8, SMARCA4, SMARCB1 |
| helicase activity | 3 | SRCAP, SMARCA2, SMARCA4 |
| neural retina development | 3 | ACTL6A, SMARCA4, SMARCD3 |
| plasma membrane | 3 | ACTL6A, ARID1B, ARID2 |
| nuclear receptor binding | 3 | ARID1A, SMARCD3, SMARCE1 |
| nuclear chromosome | 3 | PBRM1, SMARCB1, SMARCE1 |
| transcription coregulator activity | 3 | SMARCD1, SMARCD2, SMARCD3 |
| DNA-templated transcription | 3 | SMARCD1, SMARCD2, SMARCD3 |
| nuclear localization sequence binding | 2 | KPNA6, KPNA2 |
| nuclear import signal receptor activity | 2 | KPNA6, KPNA2 |
| NLS-bearing protein import into nucleus | 2 | KPNA6, KPNA2 |
| entry of viral genome into host nucleus through nuclear pore complex via importin | 2 | KPNA6, KPNA2 |
|  viral life cycle | 2 | KPNA6, KPNA2 |
| NLS-dependent protein nuclear import complex | 2 | KPNA6, KPNA2 |
| host cell | 2 | KPNA6, KPNA2 |
| identical protein binding | 2 | C17orf49, SMARCB1 |
| chromatin organization | 2 | C17orf49, SS18L1 |
| MLL1 complex | 2 | C17orf49, CHD8 |
| histone deacetylase binding | 2 | KPNA2, NIPBL |
| nuclear membrane | 2 | KPNA2, NUP50 |
| transcription corepressor activity | 2 | DPF2, SMARCA4 |
| biological_process | 2 | BRD9, BCL7B |
| ATP hydrolysis activity | 2 | CHD8, SRCAP |
| brain development | 2 | CHD8, NIPBL |
| Wnt signaling pathway | 2 | CHD8, BCL7B |
| digestive tract development | 2 | CHD8, NIPBL |
| mRNA cap binding complex binding | 2 | SRRT, PHAX |
| nuclear body | 2 | SRRT, SRCAP |
| ribonucleoprotein complex | 2 | SRRT, PHAX |
| nucleosome | 2 | SRCAP, ACTL6A |
| DNA repair | 2 | BOD1L1, ACTL6A |
| DNA damage response | 2 | BOD1L1, NIPBL |
| heart morphogenesis | 2 | NIPBL, ARID2 |
|  embryonic development | 2 | NIPBL, ACTL6A |
| integrator complex | 2 | NIPBL, INTS9 |
|  double-strand break repair via homologous recombination | 2 | ACTL6A, ARID2 |
| nucleosome binding | 2 | ARID1A, ARID1B |
| ATP-dependent activity, acting on DNA | 2 | SMARCA2, SMARCA4 |
| hydrolase activity | 2 | SMARCA2, SMARCA4 |
| negative regulation of cell growth | 2 | SMARCA2, SMARCA4 |
| RNA polymerase I core promoter sequence-specific DNA binding | 2 | SMARCA4, SMARCB1 |
| transcription coregulator binding | 2 | SMARCA4, SMARCD3 |
| Tat protein binding | 2 | SMARCA4, SMARCB1 |
| RNA polymerase I preinitiation complex assembly | 2 | SMARCA4, SMARCB1 |
| positive regulation by host of viral transcription | 2 | SMARCA4, SMARCB1 |
|  DNA-binding transcription factor activity | 2 | SMARCA4, SMARCB1 |
|  transcription of nucleolar large rRNA by RNA polymerase I | 2 | SMARCA4, SMARCB1 |
|  glucose mediated signaling pathway | 2 | SMARCA4, SMARCB1 |
| fibrillar center | 2 | SMARCA4, SMARCB1 |
| nucleolus | 2 | SMARCA4, SMARCB1 |
| XY body | 2 | SMARCB1, SMARCC1 |
| signaling receptor binding | 2 | SMARCD1, SMARCD3 |

