# Cluster2-23 Analysis

## Proteins: 

ALKBH2, B4GALT2, CGAS, FBXL6, FBXO10, FBXO33, GTF3C5, KLHDC10, KMT5C, MPG, PHIP, PIF1, PLEKHG2, PPM1D, PRIM2, RAC1, RBM43, SLF1, SLX1A, SMARCAL1, SMCR8, TAF6L, TASOR, TBPL1, TRIM33, TSEN34, WDR90, ZBTB9, ZC3H3, ZRANB3

### Nucleoplasmic DNA Repair and Chromatin Remodeling System

**Summary**

The human protein system analyzed here consists of 30 proteins, which are involved in various cellular processes, primarily related to DNA repair, chromatin remodeling, and transcription regulation. The majority of these proteins are located in the nucleoplasm (19 proteins) and nucleus (12 proteins), with some also found in the cytoplasm (8 proteins) and cytosol (6 proteins). Key biological processes performed by this system include DNA repair (6 proteins), protein ubiquitination (4 proteins), ATP binding (4 proteins), chromatin remodeling (4 proteins), and DNA damage response (4 proteins).

Based on the analysis of the proteins and their functions, this system can be named the "Nucleoplasmic DNA Repair and Chromatin Remodeling System." This name reflects the cellular location and primary functions of the proteins involved in the system.

**References**

1. UniProt Consortium. (2021). UniProt: the universal protein knowledgebase in 2021. Nucleic Acids Research, 49(D1), D480-D489. [https://doi.org/10.1093/nar/gkaa1100](https://doi.org/10.1093/nar/gkaa1100)

2. The Gene Ontology Consortium. (2021). The Gene Ontology resource: enriching a GOld mine. Nucleic Acids Research, 49(D1), D325-D334. [https://doi.org/10.1093/nar/gkaa1113](https://doi.org/10.1093/nar/gkaa1113)

3. Szklarczyk, D., Gable, A. L., Lyon, D., Junge, A., Wyder, S., Huerta-Cepas, J., ... & Mering, C. V. (2019). STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Research, 47(D1), D607-D613. [https://doi.org/10.1093/nar/gky1131](https://doi.org/10.1093/nar/gky1131)

### Validated References: 

[1] Alvarez, Silvia, da Silva Almeida, Ana C, Albero, Robert, Biswas, Mayukh, Barreto-Galvez, Angelica, Gunning, Thomas S, Shaikh, Anam, Aparicio, Tomas, Wendorff, Agnieszka, Piovan, Erich, Van Vlierberghe, Pieter, Gygi, Steven, Gautier, Jean, Madireddy, Advaitha, A Ferrando, Adolfo. "Functional mapping of PHF6 complexes in chromatin remodeling, replication dynamics, and DNA repair." Blood, 2022, pp. 3418-3429.

[2] Mazloumi Gavgani, Fatemeh, Slinning, Malene Skuseth, Morovicz, Andrea Papdiné, Arnesen, Victoria Smith, Turcu, Diana C, Ninzima, Sandra, D'Santos, Clive S, Lewis, Aurélia E. "Nuclear Phosphatidylinositol 3,4,5-Trisphosphate Interactome Uncovers an Enrichment in Nucleolar Proteins." Molecular & cellular proteomics : MCP, 2021, pp. 100102.

[3] Ptak, Christopher, Wozniak, Richard W. "Nucleoporins and chromatin metabolism." Current opinion in cell biology, 2016, pp. 153-160.

[4] Gerhold, Christian-Benedikt, Hauer, Michael H, Gasser, Susan M. "INO80-C and SWR-C: guardians of the genome." Journal of molecular biology, 2015, pp. 637-51.



## Gene Feature Summary: 

| Feature | Number of Genes | Genes |
| --- | --- | --- |
| protein binding | 25 | TBPL1, FBXO10, FBXO33, SMCR8, ZRANB3, RBM43, TRIM33, CGAS, RAC1, MPG, PPM1D, ZBTB9, ALKBH2, GTF3C5, KLHDC10, KMT5C, PHIP, PLEKHG2, PRIM2, SLF1, SLX1A, SMARCAL1, TAF6L, TASOR, WDR90 |
| nucleoplasm | 19 | SMCR8, ZRANB3, TRIM33, CGAS, MPG, PPM1D, ALKBH2, B4GALT2, GTF3C5, KLHDC10, KMT5C, PIF1, PRIM2, SLF1, SLX1A, SMARCAL1, TAF6L, TASOR, TSEN34 |
| nucleus | 12 | TRIM33, CGAS, RAC1, PPM1D, ZBTB9, ALKBH2, KMT5C, PHIP, SLF1, SMARCAL1, TAF6L, ZC3H3 |
| metal ion binding | 9 | ZRANB3, CGAS, PPM1D, ZBTB9, B4GALT2, KMT5C, PRIM2, SLX1A, ZC3H3 |
| cytoplasm | 8 | TBPL1, FBXO10, SMCR8, CGAS, RAC1, KLHDC10, SLF1, WDR90 |
| cytosol | 6 | FBXO10, CGAS, RAC1, MPG, PPM1D, PLEKHG2 |
| DNA repair | 6 | ZRANB3, CGAS, KMT5C, PIF1, SLX1A, SMARCAL1 |
| DNA binding | 6 | TRIM33, CGAS, GTF3C5, PRIM2, TAF6L, ZC3H3 |
| protein ubiquitination | 4 | FBXO10, FBXO33, TRIM33, KLHDC10 |
| ATP binding | 4 | ZRANB3, CGAS, PIF1, SMARCAL1 |
| chromatin remodeling | 4 | ZRANB3, KMT5C, SMARCAL1, TAF6L |
| DNA damage response | 4 | ZRANB3, CGAS, SLF1, SMARCAL1 |
|  transcription by RNA polymerase II | 4 | ZBTB9, PHIP, SMARCAL1, TAF6L |
| ubiquitin-protein transferase activity | 3 | FBXO10, TRIM33, FBXL6 |
| chromatin binding | 3 | CGAS, KMT5C, TASOR |
| site of double-strand break | 3 | CGAS, SLF1, SMARCAL1 |
| nucleolus | 3 | PPM1D, ALKBH2, TSEN34 |
| RNA polymerase II general transcription initiation factor activity | 2 | TBPL1, TAF6L |
| SCF-dependent proteasomal ubiquitin-dependent protein catabolic process | 2 | FBXO33, FBXL6 |
| SCF ubiquitin ligase complex | 2 | FBXO33, FBXL6 |
| guanyl-nucleotide exchange factor activity | 2 | SMCR8, PLEKHG2 |
| protein kinase binding | 2 | SMCR8, RAC1 |
| postsynapse | 2 | SMCR8, RAC1 |
| ATP-dependent DNA/DNA annealing activity | 2 | ZRANB3, SMARCAL1 |
| ATP-dependent chromatin remodeler activity | 2 | ZRANB3, SMARCAL1 |
| replication fork processing | 2 | ZRANB3, SMARCAL1 |
| nuclear replication fork | 2 | ZRANB3, SMARCAL1 |
| RNA binding | 2 | RBM43, TASOR |
| R-SMAD binding | 2 | TRIM33, ZC3H3 |
| GTP binding | 2 | CGAS, RAC1 |
| plasma membrane | 2 | CGAS, RAC1 |
| protein-containing complex binding | 2 | RAC1, SLF1 |
|  protein phosphorylation | 2 | RAC1, PHIP |
|  cell shape | 2 | RAC1, PHIP |
| extracellular exosome | 2 | RAC1, TAF6L |
| DNA dealkylation involved in DNA repair | 2 | MPG, ALKBH2 |
| negative regulation of gene expression, epigenetic | 2 | PPM1D, TASOR |
| heterochromatin | 2 | KMT5C, TASOR |
|  DNA-templated transcription | 2 | PHIP, TAF6L |
|  telomere maintenance | 2 | PIF1, SLF1 |
| chromosome, telomeric region | 2 | PIF1, SLF1 |
| double-strand break repair via homologous recombination | 2 | SLF1, SLX1A |
| centrosome | 2 | SLF1, WDR90 |
| t-circle formation | 2 | SLX1A, SMARCAL1 |

