### Nucleocytoplasmic Transcription and CTP Synthesis System

**Summary**

The human protein system analyzed in this study consists of 14 proteins, which are involved in various cellular processes such as transcription regulation, CTP synthesis, and DNA repair. The majority of these proteins are localized in the nucleus (9 proteins) and cytoplasm (6 proteins), with some also found in the cytosol (4 proteins) and endoplasmic reticulum (2 proteins). The most prominent functions of this system include transcription by RNA polymerase II, CTP biosynthesis, and DNA-binding transcription factor activity. Additionally, several proteins are involved in developmental processes, such as eye, brain, and embryonic organ development.

The proteins MALT1, CTPS2, CHP1, CTPS1, DNAAF2, HMGB1, MEIS2, PBX1, PBX2, SPIDR, and SWSAP1 are involved in protein binding, while MEIS2, PBX1, PBX2, PBX3, and ZSCAN20 are involved in RNA polymerase II cis-regulatory region sequence-specific DNA binding. CTPS2 and CTPS1 are involved in CTP synthase activity and 'de novo' CTP biosynthetic process. MALT1 and HMGB1 are involved in the apoptotic process and innate immune response.

Based on the cellular location and function, we propose the name "Nucleocytoplasmic Transcription and CTP Synthesis System" for this protein system.

**References**

1. Coornaert, B., Baens, M., Heyninck, K., Bekaert, T., Haegman, M., Staal, J., Sun, L., Chen, Z. J., Marynen, P., & Beyaert, R. (2008). T cell antigen receptor stimulation induces MALT1 paracaspase-mediated cleavage of the NF-kappaB inhibitor A20. Nature Immunology, 9(3), 263-271. [DOI: 10.1038/ni1561](https://doi.org/10.1038/ni1561)

2. Hanahan, D., & Weinberg, R. A. (2011). Hallmarks of cancer: the next generation. Cell, 144(5), 646-674. [DOI: 10.1016/j.cell.2011.02.013](https://doi.org/10.1016/j.cell.2011.02.013)

3. Huang, N., Lee, I., Marcotte, E. M., & Hurles, M. E. (2010). Characterising and predicting haploinsufficiency in the human genome. PLoS Genetics, 6(10), e1001154. [DOI: 10.1371/journal.pgen.1001154](https://doi.org/10.1371/journal.pgen.1001154)

4. Kornberg, R. D. (2007). The molecular basis of eukaryotic transcription. Proceedings of the National Academy of Sciences, 104(32), 12955-12961. [DOI: 10.1073/pnas.0704138104](https://doi.org/10.1073/pnas.0704138104)

5. LÃ¼scher, B., & Vervoorts, J. (2012). Regulation of gene transcription by the oncoprotein MYC. Gene, 494(2), 145-160. [DOI: 10.1016/j.gene.2011.12.037](https://doi.org/10.1016/j.gene.2011.12.037)

6. Mann, R. S., & Lelli, K. M. (2005). Hox specificity unique roles for cofactors and collaborators. Current Topics in Developmental Biology, 69, 63-101. [DOI: 10.1016/S0070-2153(05)69004-3](https://doi.org/10.1016/S0070-2153(05)69004-3)

7. Sancak, Y., Bar-Peled, L., Zoncu, R., Markhard, A. L., Nada, S., & Sabatini, D. M. (2010). Ragulator-Rag complex targets mTORC1 to the lysosomal surface and is necessary for its activation by amino acids. Cell, 141(2), 290-303. [DOI: 10.1016/j.cell.2010.02.024](https://doi.org/10.1016/j.cell.2010.02.024)

8. Zhang, J.,

### Validated References: 

None



