# Cluster4-16 Analysis

## Proteins: 

ANKRD11, CTC1, DBF4, EAF1, ELL2, ERCC2, ERCC3, ERCC5, GTF2H1, GTF2H3, GTF2H4, HDGFL2, MNAT1, TEN1

### Nucleoplasmic DNA Repair and Transcription System

**Summary**

The human protein system analyzed here consists of 14 proteins, which are primarily involved in DNA repair, transcription, and telomere maintenance processes. The majority of these proteins are located in the nucleoplasm and nucleus, with some also found in the cytosol and nuclear speck. The system includes proteins from the transcription factor TFIIH core and holo complexes, the CST complex, and the transcription elongation factor complex. Notably, seven proteins are involved in nucleotide-excision repair, and six proteins participate in transcription by RNA polymerase II. Based on the cellular location and function, we propose the name "Nucleoplasmic DNA Repair and Transcription System" for this protein system.

**Cellular Components and Complexes**

The proteins in this system are involved in several cellular components and complexes, including:

- Nucleoplasm (13 proteins)
- Nucleus (9 proteins)
- Transcription factor TFIIH core complex (6 proteins)
- Transcription factor TFIIH holo complex (6 proteins)
- Transcription factor TFIID complex (4 proteins)
- CST complex (2 proteins)
- Transcription elongation factor complex (2 proteins)

**Mechanisms and Biological Processes**

The Nucleoplasmic DNA Repair and Transcription System is involved in various mechanisms and biological processes, such as:

- Nucleotide-excision repair (7 proteins)
- Transcription by RNA polymerase II (6 proteins)
- Single-stranded DNA binding (3 proteins)
- Transcription elongation by RNA polymerase II (3 proteins)
- Damaged DNA binding (3 proteins)
- Metal ion binding (3 proteins)
- Transcription-coupled nucleotide-excision repair (3 proteins)
- Telomeric DNA binding (2 proteins)
- Telomere capping (2 proteins)
- Negative regulation of telomere maintenance via telomerase (2 proteins)
- Multicellular organism growth (2 proteins)
- Hematopoietic stem cell proliferation (2 proteins)
- G1/S transition of mitotic cell cycle (2 proteins)
- Response to oxidative stress (2 proteins)
- UV protection (2 proteins)
- Hair cell differentiation (2 proteins)
- Mitotic cell cycle phase transition (2 proteins)

**References**

1. Compe, E., & Egly, J. M. (2016). Nucleotide excision repair and transcriptional regulation: TFIIH and beyond. *Annual review of biochemistry*, 85, 265-290. [DOI: 10.1146/annurev-biochem-060815-014453](https://doi.org/10.1146/annurev-biochem-060815-014453)
2. Sfeir, A., & de Lange, T. (2012). Removal of shelterin reveals the telomere end-protection problem. *Science*, 336(6081), 593-597. [DOI: 10.1126/science.1218498](https://doi.org/10.1126/science.1218498)
3. Smith, E. R., & Shilatifard, A. (2010). The chromatin signaling pathway: diverse mechanisms of recruitment of histone-modifying enzymes and varied biological outcomes. *Molecular cell*, 40(5), 689-701. [DOI: 10.1016/j.molcel.2010.11.031](https://doi.org/10.1016/j.molcel.2010.11.031)

### Validated References: 

[1] Gopaul, Diyavarshini, Denby Wilkes, Cyril, Goldar, Arach, Giordanengo Aiach, Nathalie, Barrault, Marie-Bénédicte, Novikova, Elizaveta, Soutourina, Julie. "Genomic analysis of Rad26 and Rad1-Rad10 reveals differences in their dependence on Mediator and RNA polymerase II." Genome research, 2022, pp. 1516-28.

[2] Geijer, Marit E, Zhou, Di, Selvam, Kathiresan, Steurer, Barbara, Mukherjee, Chirantani, Evers, Bastiaan, Cugusi, Simona, van Toorn, Marvin, van der Woude, Melanie, Janssens, Roel C, Kok, Yannick P, Gong, Wenzhi, Raams, Anja, Lo, Calvin S Y, Lebbink, Joyce H G, Geverts, Bart, Plummer, Dalton A, Bezstarosti, Karel, Theil, Arjan F, Mitter, Richard, Houtsmuller, Adriaan B, Vermeulen, Wim, Demmers, Jeroen A A, Li, Shisheng, van Vugt, Marcel A T M, Lans, Hannes, Bernards, René, Svejstrup, Jesper Q, Ray Chaudhuri, Arnab, Wyrick, John J, Marteijn, Jurgen A. "Elongation factor ELOF1 drives transcription-coupled repair and prevents genome instability." Nature cell biology, 2021, pp. 608-619.

[3] Khanal, Pukar, Patil, Vishal S, Bhandare, Vishwambhar V, Patil, Priyanka P, Patil, B M, Dwivedi, Prarambh S R, Bhattacharya, Kunal, Harish, Darasaguppe R, Roy, Subarna. "Systems and <i>in vitro</i> pharmacology profiling of diosgenin against breast cancer." Frontiers in pharmacology, 2022, pp. 1052849.

[4] Sun, Yanan, Wang, Huihui, Qu, Taiqi, Luo, Junjie, An, Peng, Ren, Fazheng, Luo, Yongting, Li, Yixuan. "mTORC2: a multifaceted regulator of autophagy." Cell communication and signaling : CCS, 2023, pp. 4.

[5] Kolesnikova, Olga, Radu, Laura, Poterszman, Arnaud. "TFIIH: A multi-subunit complex at the cross-roads of transcription and DNA repair." Advances in protein chemistry and structural biology, 2019, pp. 21-67.

[6] Spedale, Gianpiero, Meddens, Claartje A, Koster, Maria J E, Ko, Cheuk W, van Hooff, Sander R, Holstege, Frank C P, Timmers, H Th Marc, Pijnappel, W W M Pim. "Tight cooperation between Mot1p and NC2β in regulating genome-wide transcription, repression of transcription following heat shock induction and genetic interaction with SAGA." Nucleic acids research, 2012, pp. 996-1008.

[7] Wilson, Kelly M, He, Johnny J. "HIV Nef Expression Down-modulated GFAP Expression and Altered Glutamate Uptake and Release and Proliferation in Astrocytes." Aging and disease, 2023, pp. 152-169.

[8] Säisä-Borreill, Susanna, Davidson, Guillaume, Kleiber, Thomas, Thevenot, Andréa, Martin, Elisabeth, Mondot, Stanislas, Blottière, Hervé, Helleux, Alexandra, Mengus, Gabrielle, Plateroti, Michelina, Duluc, Isabelle, Davidson, Irwin, Freund, Jean-Noel. "General transcription factor TAF4 antagonizes epigenetic silencing by Polycomb to maintain intestine stem cell functions." Cell death and differentiation, 2023, pp. 839-853.

[9] Qian, Jie, Zheng, Min, Wang, Lingyu, Song, Yu, Yan, Jiawen, Hsu, Yi-Feng. "Arabidopsis mitochondrial single-stranded DNA-binding proteins SSB1 and SSB2 are essential regulators of mtDNA replication and homologous recombination." Journal of integrative plant biology, 2022, pp. 1952-1965.

[10] Dubiel, Katarzyna, Myers, Angela R, Kozlov, Alexander G, Yang, Olivia, Zhang, Jichuan, Ha, Taekjip, Lohman, Timothy M, Keck, James L. "Structural Mechanisms of Cooperative DNA Binding by Bacterial Single-Stranded DNA-Binding Proteins." Journal of molecular biology, 2019, pp. 178-195.

[11] Osorio Garcia, Miguel A, Wood, Elizabeth A, Keck, James L, Cox, Michael M. "Interaction with single-stranded DNA-binding protein (SSB) modulates Escherichia coli RadD DNA repair activities." The Journal of biological chemistry, 2023, pp. 104773.

[12] Hayashi, Yuichiro, Funakoshi, Masafumi, Hirosawa, Kaname, Zhang-Akiyama, Qiu-Mei. "The H2TH-like motif of the Escherichia coli multifunctional protein KsgA is required for DNA binding involved in DNA repair and the suppression of mutation frequencies." Genes and environment : the official journal of the Japanese Environmental Mutagen Society, 2023, pp. 13.

[13] Schaich, Matthew A, Schnable, Brittani L, Kumar, Namrata, Roginskaya, Vera, Jakielski, Rachel C, Urban, Roman, Zhong, Zhou, Kad, Neil M, Van Houten, Bennett. "Single-molecule analysis of DNA-binding proteins from nuclear extracts (SMADNE)." Nucleic acids research, 2023, pp. e39.

[14] Melis, Joost P M, van Steeg, Harry, Luijten, Mirjam. "Oxidative DNA damage and nucleotide excision repair." Antioxidants & redox signaling, 2013, pp. 2409-19.

[15] Wijnhoven, Susan W P, Hoogervorst, Esther M, de Waard, Harm, van der Horst, Gijsbertus T J, van Steeg, Harry. "Tissue specific mutagenic and carcinogenic responses in NER defective mouse models." Mutation research, 2007, pp. 77-94.

[16] Sakasai, Ryo, Wakasugi, Mitsuo, Matsui, Tadashi, Sunatani, Yumi, Saijo, Masafumi, Matsunaga, Tsukasa, Iwabuchi, Kuniyoshi. "Camptothecin compromises transcription recovery and cell survival against cisplatin and ultraviolet irradiation regardless of transcription-coupled nucleotide excision repair." DNA repair, 2022, pp. 103318.

[17] Cai, Hong-Qing, Liu, Ang-Si, Zhang, Min-Jie, Liu, Hou-Jie, Meng, Xiao-Li, Qian, Hai-Peng, Wan, Jing-Hai. "Identifying Predictive Gene Expression and Signature Related to Temozolomide Sensitivity of Glioblastomas." Frontiers in oncology, 2020, pp. 669.

[18] Khair, Lyne, Subramanian, Lakxmi, Moser, Bettina A, Nakamura, Toru M. "Roles of heterochromatin and telomere proteins in regulation of fission yeast telomere recombination and telomerase recruitment." The Journal of biological chemistry, 2010, pp. 5327-37.

[19] Sumer, Huseyin, Nicholls, Craig, Pinto, Alexander R, Indraharan, Dinesh, Liu, Jun, Lim, Mei Ling, Liu, Jun-Ping, Verma, Paul J. "Chromosomal and telomeric reprogramming following ES-somatic cell fusion." Chromosoma, 2010, pp. 167-76.

[20] Wang, Zhiwei, Pan, Ni, Yan, Jiahao, Wan, Jian, Wan, Cuihong. "Systematic Identification of Microproteins during the Development of <i>Drosophila melanogaster</i>." Journal of proteome research, 2022, pp. 1114-1123.

[21] Dash, Sagarika, Duraivelan, Kheerthana, Samanta, Dibyendu. "Cadherin-mediated host-pathogen interactions." Cellular microbiology, 2021, pp. e13316.

[22] Li, Peiyao, Jing, Huiru, Wang, Yanzhe, Yuan, Lei, Xiao, Hui, Zheng, Qian. "SUMO modification in apoptosis." Journal of molecular histology, 2021, pp. 1-10.



## Gene Feature Summary: 

| Feature | Number of Genes | Genes |
| --- | --- | --- |
| protein binding | 13 | CTC1, GTF2H4, EAF1, DBF4, ELL2, TEN1, HDGFL2, ERCC2, ERCC3, ERCC5, GTF2H1, GTF2H3, MNAT1 |
| nucleoplasm | 13 | CTC1, GTF2H4, EAF1, DBF4, ELL2, TEN1, ANKRD11, ERCC2, ERCC3, ERCC5, GTF2H1, GTF2H3, MNAT1 |
| nucleus | 9 | CTC1, GTF2H4, TEN1, ANKRD11, HDGFL2, ERCC2, ERCC3, ERCC5, GTF2H3 |
| nucleotide-excision repair | 7 | GTF2H4, ERCC2, ERCC3, ERCC5, GTF2H1, GTF2H3, MNAT1 |
| DNA repair | 6 | GTF2H4, HDGFL2, ERCC3, GTF2H1, GTF2H3, MNAT1 |
| transcription by RNA polymerase II | 6 | GTF2H4, ERCC2, ERCC3, GTF2H1, GTF2H3, MNAT1 |
| transcription factor TFIIH core complex | 6 | GTF2H4, ERCC2, ERCC3, GTF2H1, GTF2H3, MNAT1 |
| transcription factor TFIIH holo complex | 6 | GTF2H4, ERCC2, ERCC3, GTF2H1, GTF2H3, MNAT1 |
| transcription factor TFIID complex | 4 | GTF2H4, ERCC2, ERCC3, GTF2H3 |
| single-stranded DNA binding | 3 | CTC1, TEN1, ERCC5 |
| cytosol | 3 | CTC1, ANKRD11, ERCC2 |
| transcription elongation by RNA polymerase II | 3 | EAF1, ELL2, ERCC3 |
| damaged DNA binding | 3 | ERCC2, ERCC3, ERCC5 |
| metal ion binding | 3 | ERCC2, ERCC5, GTF2H3 |
| transcription-coupled nucleotide-excision repair | 3 | ERCC2, ERCC3, ERCC5 |
| telomeric DNA binding | 2 | CTC1, TEN1 |
| telomere capping | 2 | CTC1, TEN1 |
| negative regulation of telomere maintenance via telomerase | 2 | CTC1, TEN1 |
| multicellular organism growth | 2 | CTC1, ERCC2 |
| hematopoietic stem cell proliferation | 2 | CTC1, ERCC2 |
| chromosome, telomeric region | 2 | CTC1, TEN1 |
| CST complex | 2 | CTC1, TEN1 |
| double-stranded DNA binding | 2 | GTF2H4, ERCC5 |
| RNA polymerase II general transcription initiation factor activity | 2 | GTF2H4, GTF2H3 |
| core TFIIH complex portion of holo TFIIH complex | 2 | GTF2H4, GTF2H3 |
| nuclear speck | 2 | GTF2H4, EAF1 |
|  transcription elongation by RNA polymerase II | 2 | EAF1, ELL2 |
| transcription elongation factor complex | 2 | EAF1, ELL2 |
| nuclear body | 2 | EAF1, DBF4 |
| enzyme activator activity | 2 | DBF4, ERCC5 |
| zinc ion binding | 2 | DBF4, MNAT1 |
| G1/S transition of mitotic cell cycle | 2 | DBF4, MNAT1 |
| cytoplasm | 2 | HDGFL2, ERCC2 |
| ATP binding | 2 | ERCC2, ERCC3 |
| ATP hydrolysis activity | 2 | ERCC2, ERCC3 |
| apoptotic process | 2 | ERCC2, ERCC3 |
| response to oxidative stress | 2 | ERCC2, ERCC3 |
| UV protection | 2 | ERCC2, ERCC3 |
| hair cell differentiation | 2 | ERCC2, ERCC3 |
|  mitotic cell cycle phase transition | 2 | ERCC2, ERCC3 |
| CAK-ERCC2 complex | 2 | ERCC2, MNAT1 |
| response to UV | 2 | ERCC3, ERCC5 |
| transcription preinitiation complex | 2 | ERCC3, GTF2H3 |
| negative regulation of apoptotic process | 2 | ERCC5, MNAT1 |
|  cyclin-dependent protein serine/threonine kinase activity | 2 | GTF2H1, MNAT1 |
|  DNA-templated transcription | 2 | GTF2H1, GTF2H3 |

